enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,244 @@
+---
+title: rhel-8-server-worker-infra-node-support
+authors:
+  - "@mtnbikenc"
+reviewers:
+  - "@staebler"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2021-05-13
+last-updated: 2021-06-23
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CORS-1650"
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# Traditional RHEL 8 Server Worker/Infra Node Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add traditional RHEL 8 Server support for deploying worker/infra
+nodes with [openshift-ansible].  OpenShift will be deployed and upgraded on RHEL 8 nodes using
+the same [Ansible playbook process][adding-rhel-compute].
+
+## Motivation
+
+Using the [existing process][adding-rhel-compute] will allow near-term support
+of RHEL 8 nodes while developing enhanced longer-term RHEL 8 support.
+
+### Goals
+
+1. Update [openshift-ansible] scaleup.yml and upgrade.yml playbooks to support traditional RHEL 8 workers
+
+### Non-Goals
+
+* Deploying/upgrading traditional RHEL 8 worker nodes through other mechanisms, i.e. operators
+* Changing the deployment/upgrade process for RHEL 7 worker nodes
+* Upgrading existing RHEL 7 worker nodes to RHEL 8
+* OKD and traditional Fedora worker support
+
+## Proposal
+
+The basic process for scaling up and upgrading RHEL worker nodes will remain the
+same.  The playbooks will be updated to install the appropriate RPMs based on
+the OS during the install/upgrade for RHEL 8 hosts.  The installation process will
+not change for RHEL 7 worker nodes.
+
+To migrate from RHEL 7 worker nodes, new RHEL 8 nodes should be provisioned,
+and the existing RHEl 7 worker nodes removed.
+
+
+### User Stories
+
+1. As an OpenShift admin using [openshift-ansible], I want to deploy and upgrade
+   traditional RHEL 8 Server worker nodes.
```

Comment:
What platforms do customers tend to want this for? All? Metal? Specific clouds? It probably doesn't matter in terms of this doc, but it helps add context.